src/ucp/wireup/wireup.c

```diff
@@ -791,6 +797,17 @@ ucp_wireup_connect_lane_to_iface(ucp_ep_h ep, ucp_lane_index_t lane,
         return UCS_ERR_UNREACHABLE;
     }
 
+    if ((ep->uct_eps[lane] == NULL) && ucp_ep_has_cm_lane(ep)) {
+        /* create wireup EP in case of CM lane is used, since a WIREUP EP is
+         * used to keep user's pending requests and send WIREUP MSGs (if this
+         * is WIREUP MSG lane) until CM and WIREUP_MSG phases are not done */
```

Comment:
minor: `until ... are done` or `while ... are not done`

Reply:
done